src/GraphQL/InputProcessor/Base.php

```diff
@@ -72,7 +72,7 @@ public function process(Concrete $object, $newValue, $args, $context, ResolveInf
      */
     public function getParentProcessor($nodeDef, ClassDefinition $class) {
         $nodeDefAttributes = $nodeDef["attributes"];
-        $children = $nodeDefAttributes["childs"];
+        $children = $nodeDefAttributes['children'];
```

Comment:
I fear that this is not possible because we would also have to change it on the client side which could have a huge impact.

Reply:
see fieldConfigDialog.js (which is more or a less a copy of gridConfigDialog.js)

Reply:
I could have sworn I tested it (after clearing caches) and it worked, but you are correct, I am reverting these two lines